modules/contact-form/css/grunion.css

```diff
@@ -105,12 +103,3 @@
 	display: inline-block;
 	margin-top: 7px;
 }
-
-@media only screen and (min-width: 600px) {
-	.contact-form input[type='text'],
-	.contact-form input[type='email'],
-	.contact-form input[type='tel'],
-	.contact-form input[type='url'] {
-		width: 50%;
-	}
```

Comment:
I am a bit worried about those changes. Contact forms have been used in Jetpack for a long time, and folks have customized the look of their forms to make them fit on pages, sidebars, home pages. Making such drastic changes to all forms at once may have a bad impact on existing forms.

Could we instead add a more specific rule to only impact Form blocks? It will most likely impact a lot less people that way.

Reply:
Yes this makes sense. I've added a new classname to the form when the block has been used and instead adjusted the width based on the presence of this class.